build.gradle.kts

```diff
@@ -33,6 +33,28 @@ allprojects {
 	}
 }
 
+fun DependencyHandlerScope.addDatabaseDependencies() {
+	// https://mvnrepository.com/artifact/mysql/mysql-connector-java
+	runtimeOnly("mysql:mysql-connector-java:8.0.32")
+	/*
+		LINE Lib versions - https://github.com/line/kotlin-jdsl/blob/main/libs.versions.toml
+		Kotlin JDSL with Spring Boot 3.x version
+	 	    https://github.com/line/kotlin-jdsl/blob/main/spring/data-reactive-core/README.md
+	 	Kotlin JDSL Examples
+	 	    https://github.com/cj848/kotlin-jdsl-example
+	 */
+	implementation("org.springframework.boot:spring-boot-starter-data-jpa") {
+		exclude(module = "hibernate-core")
+		exclude(module = "hibernate-commons-annotations")
+	}
+	implementation("com.linecorp.kotlin-jdsl:spring-data-kotlin-jdsl-hibernate-reactive-jakarta:2.2.1.RELEASE")
+	implementation("org.hibernate.reactive:hibernate-reactive-core-jakarta:1.1.9.Final")
+	implementation("io.vertx:vertx-jdbc-client:4.3.1")
+	implementation("io.agroal:agroal-pool:2.0")
+	implementation("io.smallrye.reactive:mutiny-kotlin:1.6.0")
```

Comment:
2.2.0 version is up to date, should I use 1.6.0?

Reply:
Updated 2.2.0